src/aat/java/uk/gov/hmcts/cmc/claimstore/tests/helpers/CommonOperations.java

```diff
@@ -50,11 +59,25 @@ private Response saveClaim(ClaimData claimData, String userAuthentication, Strin
             .post("/claims/" + userId);
     }
 
-    public void linkDefendant(String claimExternalId, String userAuthentication, String userId) {
+    public void testCasesRetrievalFor(String uriPath) {
+        String response = RestAssured
+            .given()
+            .header(HttpHeaders.AUTHORIZATION, bootstrap.getSmokeTestCitizen().getAuthorisation())
+            .when()
+            .get(uriPath)
+            .then()
+            .statusCode(HttpStatus.OK.value())
+            .and()
+            .extract().body().asString();
+
+        assertThat(response).matches(jsonListPattern);
```

Comment:
What this test does prove? Assertion will be satisfied no matter whether empty or populated list returned.